components/brave_wallet_ui/common/slices/endpoints/pricing.endpoints.ts

```diff
@@ -279,7 +279,7 @@ export const pricingEndpoints = ({
                 aggregatedBalances[chainId] = { tokenBalances: {} }
               }
 
-              const chainBalances = aggregatedBalances[chainId]
+              const chainBalances = aggregatedBalances[chainId].tokenBalances
```

Comment:
I've cherry-picked the fix for uplift to 1.64 https://github.com/brave/brave-core/pull/22141
Thanks!